test/image/mocks/marker_symbols.json

```diff
@@ -0,0 +1,99 @@
+{
+  "data": [
+    {
+      "x": [1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9],
+      "y": [1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 4, 4, 4, 4, 4, 4, 4, 4, 4, 5, 5, 5, 5, 5, 5, 5, 5, 5],
+      "mode": "markers",
+      "text": [
+        "marker number: 0<br>marker string: 'circle'",
```

Comment:
Done in https://github.com/plotly/plotly.js/pull/1781/commits/083b00a9832211dfa1f02e511fdaf8698fedce1d